core/services/streams/stream_registry.go

```diff
@@ -0,0 +1,64 @@
+package streams
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/pipeline"
+)
+
+type StreamID string
+
+func (s StreamID) String() string {
+	return string(s)
+}
```

Comment:
How about an alias? You will be able to drop trival conversions too, like `StreamID("test")`.
```suggestion
type StreamID = string
```